routes/members.js

```diff
@@ -284,4 +291,39 @@ router.patch("/delete-isMember", authenticate, authorizeUser("superUser"), membe
 
 router.patch("/archiveMembers/:username", authenticate, authorizeUser(SUPER_USER), members.archiveMembers);
 
+/**
+ * @swagger
+ * /members/add-default-archived-role:
+ *  patch:
+ *   summary: One time call to add default archived role for the users only where it does not exists
+ *   tags:
+ *     - Members
+ *   responses:
+ *     200:
+ *       description: Details of the users migrated
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/migratedUsers'
+ *     401:
+ *       description: unAuthorized
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/unAuthorized'
+ *     403:
+ *       description: forbidden
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/forbidden'
+ *     500:
+ *       description: badImplementation
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/badImplementation'
+ */
+router.patch("/add-default-archived-role", authenticate, authorizeUser("superUser"), members.addDefaultArchivedRole);
```

Comment:
Nit: Naming can be improved. But since this is a temporary API, we can let this one slide in.

Do you have a cleanup ticket ready for it?

Reply:
Cleanup ticket for what?

Reply:
https://github.com/Real-Dev-Squad/website-backend/issues/671